llvm_mode/afl-clang-fast.c

```diff
@@ -128,8 +128,10 @@ static void edit_params(u32 argc, char** argv) {
 
 #ifdef USE_TRACE_PC
   cc_params[cc_par_cnt++] = "-fsanitize-coverage=trace-pc-guard";
+#ifndef __ANDROID__
```

Comment:
WHy disable this on android?

Reply:
Android toolchain doesn't work with the flag, will facing build error.

Reply:
this fails in the Android toolchain because these options are not supported in modern llvm anymore.

Reply:
Maybe mention this in a comment?